### ISSUE TITLE
Add custom layout to Post carousel module

### ISSUE THREAD

**Issue by Bobolinos** [beaverbuilder/feature-requests#221]
**Is your feature request related to a problem? Please describe.**
I am not able to have a custom layout in the Post carousel module, but I can have one in the normal post module. 
A clients asks me if we can turn the normal post module into a carousel, but this is not possible simply because of this missing feature.

**Describe the solution you'd like**
Add custom layout to Post carousel module just like in the normal post module.

**Describe alternatives you've considered**
Create my own module, but I don't think thats fair for this simple feature.

**Comment by A-Buzon** [comment]
I was just coming on here to request this too!  Actually for the **Post Slider module**  (just a Post Carousel that displays multiple posts within the viewport for the most part).
I was _quite_ _surprised_ when I realized that I could NOT customize the layout!  I just assumed that tech was available anywhere a post was being displayed - to be able to rearrange the layout of the HTML, especially given our crowd's heavy usage of CPTs, Pods, ACFs.

**Comment by gbissland** [comment]
Yes, this would be useful in core.
@A-Buzon @Bobolinos  In the mean time the PowerPack Content Grid post module has a Carousel layout with custom layout option.

**Comment by zackpyle** [comment]
@Bobolinos I agree. This module is almost useless to advanced users without this feature. Like @gbissland said, using something like PP Content Grid set to Carousel is the only viable option currently

**Comment by zackpyle** [comment]
[Based on this blog](https://www.wpbeaverbuilder.com/beaver-builder-dev-update-latest-progress-and-future-plans/#slider-containers), we can move this feature request to Planned?

**Comment by zackpyle** [comment]
@codente [Based on this blog](https://www.wpbeaverbuilder.com/beaver-builder-dev-update-latest-progress-and-future-plans/#slider-containers), we can move this feature request to Planned?
